text/0000-keep-key-logs.md

```diff
@@ -0,0 +1,63 @@
+# RFC: Keep Key Logs
+
+- RFC PR: 
+- Tracking issue:
+
+## Motivation
+
+To address https://github.com/tikv/tikv/issues/10841, we decide to prioritize availability over log completeness thus switch the overflow strategy of logger from `Block` to `Drop`. 
+
+However, some logs are useful for diagnosis in practice. We heavily depend on logs to investigate bugs or abnormalities of the system. If all logs are dropped randomly, reasoning about the raftstore or transaction issues may be impossible.
+
+Therefore, it is only a temporary workaround to simply drop overflowed logs. We hope to keep logs that are important for diagnosis if possible.
+
+## Detailed design
+
+The problem is that we don't define which logs are important. Whether a log is important for diagnosis is not related to the log level. 
+
+For example, we prefer to keep info level logs in the raftstore rather than keep warn level logs in the coprocessor module. The former one is critical for finding out how each region evolves while the latter one may be only a retryable error.
+
+So, these important logs deserve to have another set of macros:
```

Comment:
Like slow log, maybe we can use tag to identify whether a logs is key log.

Reply:
Yes. As said in the "Changes to slog modules" section, these macros are just shotcuts to hide the magic tag. So it would be easier for users.